Resources/Prototypes/Entities/Structures/Furniture/chairs.yml

```
@@ -19,6 +19,9 @@
         density: 100
         mask:
         - TableMask
+        layer:
+        - TableLayer
+        - BulletImpassable
```

```suggestion
        # WD EDIT START
        layer:
        - BulletImpassable
        # WD EDIT END
```
Вот причина провала тестов. Лейер стола тут не нужен и он не давал людям сесть на стул